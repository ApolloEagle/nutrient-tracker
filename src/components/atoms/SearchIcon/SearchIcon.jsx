import React from 'react';
import { Search } from '@mui/icons-material';

const SearchIcon = () => <Search />;

export default SearchIcon;
