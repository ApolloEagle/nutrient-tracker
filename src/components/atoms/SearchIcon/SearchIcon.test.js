import React from 'react'
import { shallow } from 'enzyme';
import SearchIcon from "./SearchIcon";
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

const wrapper = shallow(<SearchIcon />);

describe("Search Icon", () => {
  it("Renders search icon", () => {
    expect(wrapper).toMatchSnapshot();
  });
});