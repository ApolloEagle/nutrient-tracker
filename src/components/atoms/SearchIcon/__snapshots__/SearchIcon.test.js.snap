// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Icon Renders search icon 1`] = `<Memo(ForwardRef(SearchIcon)) />`;
